projects/gfbio-app/src/app/app.component.ts

```diff
@@ -0,0 +1,266 @@
+            // {
+            //     name: 'Species Occurrences',
+            //     description: 'Query data from GBIF',
+            //     iconSrc: GFBioSourceType.ICON_URL,
+            //     sidenavConfig: {component: GbifOperatorComponent, keepParent: true},
+            // },
+            // SourceOperatorListComponent.createCountryPolygonsButton(),
+        ];
+    }
+
+    private static createOperatorListButtons(): OperatorListButtonGroups {
+        return [
+            {name: 'Mixed', list: OperatorListComponent.DEFAULT_MIXED_OPERATOR_DIALOGS},
+            {name: 'Plots', list: OperatorListComponent.DEFAULT_PLOT_OPERATOR_DIALOGS},
+            {name: 'Raster', list: OperatorListComponent.DEFAULT_RASTER_OPERATOR_DIALOGS},
+            {name: 'Vector', list: OperatorListComponent.DEFAULT_VECTOR_OPERATOR_DIALOGS},
+        ];
+    }
+
+    @HostListener('window:resize')
+    private windowHeight(): void {
+        this.windowHeight$.next(window.innerHeight);
+    }
+
+    // private handleQueryParameters() {
+    //     this.activatedRoute.queryParams.subscribe(p => {
+    //         for (const parameter of Object.keys(p)) {
+    //             const value = p[parameter];
+    //             switch (parameter) {
+    //                 case 'workflow':
+    //                     try {
+    //                         const newLayer = Layer.fromDict(JSON.parse(value));
+    //                         this.projectService.getProjectStream().pipe(first()).subscribe(project => {
+    //                             if (project.layers.length > 0) {
+    //                                 // show popup
+    //                                 this.dialog.open(WorkflowParameterChoiceDialogComponent, {
+    //                                     data: {
+    //                                         dialogTitle: 'Workflow URL Parameter',
+    //                                         sourceName: 'URL parameter',
+    //                                         layers: [newLayer],
+    //                                         nonAvailableNames: [],
+    //                                         numberOfLayersInProject: project.layers.length,
+    //                                     }
+    //                                 });
+    //                             } else {
+    //                                 // just add the layer if the layer array is empty
+    //                                 this.projectService.addLayer(newLayer);
+    //                             }
+    //                         });
+    //                     } catch (error) {
+    //                         this.notificationService.error(`Invalid Workflow: »${error}«`);
+    //                     }
+    //                     break;
+    //                 default:
+    //                     this.notificationService.error(`Unknown URL Parameter »${parameter}«`);
+    //             }
+    //         }
+    //     });
+    // }
```

Comment:
I guess that is a separate task re-activating this. But we should add it somewhere.

Reply:
I created a card https://trello.com/c/uMQhmiPT